src/adapters/discord.js

```diff
@@ -24,52 +24,78 @@ discordAdapter.__INIT__ = function ({ process }) {
         }
 
         const handle = (context) => {
-            const { output, reactions } = context;
-
+            const { output, reactions, outputRich } = context;
             // TODO rework!
-            if (!isEmpty(reactions)) {
-                reactions.reduce(
-                    (prev, reaction) => prev.then(() => msg.react(reaction).catch((e) => { console.error(e.message); })),
-                    Promise.resolve(),
-                );
-            }
-
-            if (isEmpty(output)) {
-                return;
-            }
-
-            // TODO: need check permissions!
-            // simple case, output as answer
-            if (isString(output)) {
+            console.log('discord: ', outputRich);
+
+            if (outputRich) {
```

Comment:
yeah, same changes in 2 separate reviews ^^

Reply:
I needed to implement it differently?

Reply:
I dunno, maybe, we need to merge first, and then create second, cause now it's hard to separate actual changes <_>